api/openapi-spec/v1.0.yaml

```diff
@@ -91,6 +91,56 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  /v1.0/me/photo:
+    get:
+      tags:
+        - me.photo
+      summary: Get the current user's profile photo
+      operationId: GetOwnUserPhoto
+      responses:
+        '200':
+          description: Retrieved entity
+          content:
+            image/png:
+              schema:
+                type: string
+                format: binary
+        default:
+          $ref: '#/components/responses/error'
+    put:
+      tags:
+        - me.photo
+      summary: Update the current user's profile photo
+      operationId: UpdateOwnUserPhotoPut
+      requestBody:
+        description: New user photo
+        content:
+          image/png:
```

Comment:
I don't know if it is restricted to that be the MS Graph docs mention `image/jpeg` here.